tests/baselines/reference/recursiveTypeReferences1.errors.txt

```diff
@@ -60,7 +60,7 @@ tests/cases/conformance/types/typeRelationships/recursiveTypes/recursiveTypeRefe
     
     const b20: Box2 = 42;  // Error
           ~~~
-!!! error TS2322: Type '42' is not assignable to type 'Box2'.
+!!! error TS2322: Type '42' is not assignable to type 'Box<number | Box2>'.

```

Comment:
:( Is it possible we want to keep the pre-normalized form around for error reporting? Since it might have a nicer alias for us to use?

Reply:
I looked at doing that, but I'm not sure it really is all that valuable. We still display the alias in all other situations, it's only when we're elaborating into the type that we now display structure one level earlier that we would before. I think the few places it affects are just as well served with the change in this PR (which, BTW, is what we used to always do).

Reply:
I know, but we acknowledged that these errors were _better_ - a nice side effect of the recursive alias change. it would be nice if we could keep them.

Reply:
Ok, changed the code to preserve the original type and use it in error reporting if it has an alias.